master/buildbot/db/model.py

```diff
@@ -759,6 +759,17 @@ def thd(engine):
             return db_version == repo_version
         return self.db.pool.do_with_engine(thd)
 
+    def enableForeignKeys(self):
+        # foreign keys is a recent feature of sqlite3, it is still disabled by default
+        # https://www.sqlite.org/pragma.html#pragma_foreign_key_check
```

Comment:
From the docs, it looks like yes.  In that case, should this function catch the exception and log it, if the support is not present?